variables.tf

```diff
@@ -834,7 +834,13 @@ variable "net_profile_outbound_type" {
 variable "net_profile_pod_cidr" {
   type        = string
   default     = null
-  description = " (Optional) The CIDR to use for pod IP addresses. This field can only be set when network_plugin is set to kubenet. Changing this forces a new resource to be created."
+  description = "(Optional) The CIDR to use for pod IP addresses. This field can only be set when network_plugin is set to kubenet. Changing this forces a new resource to be created."
+}
+
+variable "net_profile_pod_cidrs" {
```

Comment:
What happens if both `net_profile_pod_cidr` and `net_profile_pod_cidrs` are set ?